app/src/main/res/drawable/nekogram_x.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+
+    <path
+        android:fillColor="#000"
+        android:pathData="M20.07 13.13a1.133 0.954 54.911 0 1-0.13 1.476 1.133 0.954 54.911 0 1-1.432-0.379 1.133 0.954 54.911 0 1 0.13-1.475 1.133 0.954 54.911 0 1 1.431 0.379 zm-5.278 4.138a1.133 0.954 54.911 0 1-0.13 1.475 1.133 0.954 54.911 0 1-1.431-0.378 1.133 0.954 54.911 0 1 0.13-1.476 1.133 0.954 54.911 0 1 1.431 0.379 zm-3.854 4.689a10.03 10.03 0 0 1-5.827-2.705C1.58 15.873 0.982 10.47 3.689 6.423 5.055 4.38 7.182 2.88 9.542 2.291c0.9-0.224 1.463-0.291 2.462-0.291 1.035 0 1.589 0.07 2.542 0.318 4.918 1.283 8.089 6.096 7.346 11.153a9.985 9.985 0 0 1-5.51 7.517 9.593 9.593 0 0 1-3.196 0.95 c-0.642 0.075 -1.676 0.083 -2.248 0.019 zm1.953-0.228c1.567-0.153 3.157-0.708 4.441-1.55a9.71 9.71 0 0 0 3.43-3.9c0.695-1.428 0.981 -2.683 0.98 -4.292-0.001-1.202-0.148-2.095-0.52-3.178-0.296-0.857-0.391-1.03-0.663-1.202-0.95-0.6-2.79-1.371-3.466-1.451-0.304-0.037-0.363 0.033 -0.363 0.424 0.001 0.538 0.218 2.189 0.49 3.725 0.085 0.482 0.103 0.684 0.068 0.75-0.026 0.049 -0.282 0.212 -0.57 0.363 -1.09 0.574 -1.466 0.831 -2.252 1.539-0.4 0.36 -0.449 0.39 -0.626 0.387 -0.303-0.005-0.637-0.158-1.605-0.733-1.295-0.77-1.846-1.022-2.347-1.075-0.312-0.033-0.46 0.008 -0.5 0.138 -0.07 0.239 -0.106 1.47-0.069 2.43 0.066 1.712 0.103 1.908 0.665 3.487 0.34 0.956 0.347 0.986 0.487 2.014 0.136 0.997 0.254 1.498 0.455 1.927 0.073 0.155 0.105 0.178 0.29 0.202 0.35 0.047 1.169 0.045 1.675-0.005z" />
+</vector>
```

Comment:
```suggestion
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="24dp"
    android:height="24dp"
    android:viewportWidth="24"
    android:viewportHeight="24">
  <path
      android:pathData="M12,2A10,10 0,0 0,2 12,10 10,0 0,0 12,22 10,10 0,0 0,22 12,10 10,0 0,0 12,2ZM12.0008,2.7895a9.2105,9.2105 0,0 1,7.5411 3.9272l-0.014,-0.0093c0.3376,0.4854 0.6312,0.9558 0.5701,0.9558 -0.0326,0 -0.3003,-0.1221 -0.5948,-0.2714C19.2088,7.2426 18.6845,7.0298 18.338,6.9192 17.1107,6.5271 17.0325,6.6157 17.192,8.2175c0.1172,1.1765 0.3541,2.5955 0.4895,2.9313 0.0557,0.1382 -0.0527,0.2401 -0.4887,0.4603 -0.7063,0.3567 -1.3549,0.7923 -2.0471,1.3744 -0.298,0.2506 -0.5884,0.4556 -0.6451,0.4556 -0.0568,0 -0.6873,-0.3011 -1.4011,-0.6692 -0.7138,-0.3681 -1.5671,-0.7774 -1.8964,-0.9097 -0.9053,-0.3639 -0.9241,-0.3296 -0.9241,1.6852 0,1.8816 0.1541,2.8934 0.6277,4.1231 0.1729,0.4489 0.3836,1.2524 0.4683,1.7854 0.0848,0.533 0.1986,1.1463 0.2531,1.3629l0.0991,0.3937 -0.0156,-0.0208A9.2105,9.2105 0,0 1,2.7903 12,9.2105 9.2105,0 0,1 12.0008,2.7894ZM19.4702,12.8199c0.3014,-0.0098 0.6079,0.1454 0.8045,0.4161 0.3068,0.4223 0.2512,0.9857 -0.1242,1.2584 -0.3754,0.2727 -0.9284,0.1515 -1.2352,-0.2708 -0.307,-0.4223 -0.2514,-0.9858 0.124,-1.2586 0.0893,-0.0649 0.192,-0.1093 0.3024,-0.1308 0.0425,-0.0083 0.0854,-0.013 0.1285,-0.0144zM14.5508,16.6896c0.3013,-0.0097 0.6077,0.1455 0.8043,0.4161 0.3068,0.4223 0.2512,0.9857 -0.1242,1.2584 -0.3754,0.2727 -0.9284,0.1515 -1.2352,-0.2708 -0.3069,-0.4224 -0.2513,-0.9859 0.1242,-1.2586 0.0893,-0.0649 0.192,-0.1093 0.3024,-0.1308 0.0425,-0.0083 0.0855,-0.013 0.1285,-0.0144z"
      android:strokeLineJoin="round"
      android:strokeWidth="0.210526"
      android:fillColor="#000000"
      android:strokeLineCap="round"/>
</vector>
```

Reply:
I have already designed like this and I was just aiming for solid icons so it would look consistent with google app icons. One variant look good on dark theme and another one looks good on light theme. Is it possible to keep both variants and let user to choose between them?

Reply:
I’m afraid that isn’t possible, but on second thought, it does look better for the circle to be filled. Please disregard my comment above. My only suggestion would be to make the circular outline a bit thicker.

```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   width="24"
   height="24"
   version="1.1"
   id="svg4"
   sodipodi:docname="nekogram_x.svg"
   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <defs
     id="defs8" />
  <sodipodi:namedview
     id="namedview6"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="false"
     inkscape:zoom="16"
     inkscape:cx="1.125"
     inkscape:cy="13.03125"
     inkscape:window-width="2560"
     inkscape:window-height="1015"
     inkscape:window-x="0"
     inkscape:window-y="32"
     inkscape:window-maximized="1"
     inkscape:current-layer="svg4" />
  <path
     id="path2"
     style="fill:#000000;stroke-width:0.97561"
     d="M 12,2 C 6.4800469,2 2,6.4800469 2,12 2,17.519953 6.4800469,22 12,22 17.519953,22 22,17.519953 22,12 22,6.4800469 17.519953,2 12,2 Z m 8.626143,5.9889473 c 0.567249,1.2188336 0.886052,2.5772607 0.886052,4.0110477 0,5.256324 -4.255871,9.512195 -9.512195,9.512195 -0.302018,0 -0.600785,-0.01634 -0.89558,-0.04382 -1.4409642,-0.228491 -1.3499317,-0.178891 0,0 -0.07037,-0.02459 -0.104192,-0.06715 -0.152439,-0.169589 -0.196097,-0.418536 -0.311295,-0.906127 -0.443978,-1.87881 -0.136585,-1.002927 -0.142759,-1.031875 -0.474466,-1.964558 -0.5482931,-1.540488 -0.5853815,-1.732957 -0.6497717,-3.403201 -0.036098,-0.936586 3.053e-4,-2.137257 0.068597,-2.370428 0.039024,-0.126829 0.1834147,-0.167485 0.4878049,-0.135289 0.4887808,0.05171 1.0250768,0.298704 2.2884918,1.049924 0.944389,0.560975 1.270701,0.70968 1.56631,0.714558 0.172683,0.0029 0.221417,-0.02607 0.611661,-0.377287 0.76683,-0.690732 1.133614,-0.941524 2.197028,-1.501524 0.280975,-0.147317 0.52913,-0.306617 0.554496,-0.354421 0.03415,-0.06439 0.01814,-0.261465 -0.06479,-0.731707 -0.265366,-1.4985371 -0.477302,-3.1088873 -0.478277,-3.6337653 0,-0.2860976 0.03201,-0.3954892 0.173399,-0.4173018 l 5e-6,-1.3e-6 c 0.04713,-0.00727 0.106875,-0.00521 0.181021,0.00381 0.659512,0.078049 2.453507,0.8304116 3.380336,1.4157775 0.114346,0.072307 0.19657,0.1484079 0.276295,0.2743902 m -1.665396,4.5998487 c -0.04849,-4.83e-4 -0.09701,0.0021 -0.144817,0.0096 -0.124054,0.01937 -0.239296,0.06534 -0.339177,0.13529 -0.420699,0.294962 -0.477845,0.938917 -0.127668,1.438643 0.350835,0.499409 0.976088,0.66489 1.396723,0.369665 0.420699,-0.294962 0.477845,-0.938918 0.127667,-1.438644 h -0.002 c -0.224628,-0.319483 -0.571352,-0.511139 -0.910777,-0.514515 z m -5.148628,4.033917 c -0.04857,-4.03e-4 -0.09695,0.0039 -0.144818,0.01143 -0.124053,0.01937 -0.239295,0.06534 -0.339176,0.13529 -0.421882,0.294913 -0.479081,0.940326 -0.127667,1.440548 0.351369,0.498909 0.976569,0.663524 1.396722,0.36776 0.420699,-0.294963 0.477845,-0.938918 0.127668,-1.438644 -0.225023,-0.320113 -0.572766,-0.513565 -0.912729,-0.516387 z"
     sodipodi:nodetypes="ssssscsscccccccccccccccsccccccccccsccccccsccc" />
</svg>
```

Reply:
Looks great!